package.json

```diff
@@ -46,12 +46,13 @@
     ],
     "testPathIgnorePatterns": [
       "/node_modules/",
+      "<rootDir>/integration-tests/create-react-app",
       "<rootDir>/packages/react-percy-webpack/src/entry/__tests__/test-project/"
     ]
   },
   "lint-staged": {
     "*.js": [
-      "eslint --fix --color --no-ignore",
```

Comment:
@joscha added it with a [comment here](https://github.com/percy/react-percy/pull/50#discussion_r131067403) saying anything below ./storybook (in the storybook integration tests) was being ignored without this flag.  I don't have more context beyond that.